test/test_nn.py

```diff
@@ -6518,6 +6518,12 @@ def func(*inputs):
         dummy_out = func(*inputs)
         grad_y = torch.randn_like(dummy_out, device=device, dtype=dtype, requires_grad=True)
 
+        # Issue #15353: test mkldnn double backward, don't run gradgradcheck due
```

Comment:
the imprecision issues for float double-backward are expected. this looks good to me.